code/modules/overmap/objects/dynamic_datum.dm

```diff
@@ -80,9 +83,13 @@
 
 /datum/overmap/dynamic/post_docked(datum/overmap/ship/controlled/dock_requester)
 	if(planet_name)
-		for(var/mob/M as anything in GLOB.player_list)
-			if(dock_requester.shuttle_port.is_in_shuttle_bounds(M))
-				M.play_screen_text("<span class='maptext' style=font-size:24pt;text-align:center valign='top'><u>[planet_name]</u></span><br>[station_time_timestamp_fancy("hh:mm")]")
+		for(var/mob/Mob as anything in GLOB.player_list)
+			if(dock_requester.shuttle_port.is_in_shuttle_bounds(Mob))
+				Mob.play_screen_text("<span class='maptext' style=font-size:24pt;text-align:center valign='top'><u>[planet_name]</u></span><br>[station_time_timestamp_fancy("hh:mm")]")
+				if (landing_sound == "friendly")
+					playsound(Mob, 'sound/effects/planet_landing_1.ogg', 100)
```

Comment:
you should make the landing_sound variable just the path to the sound, and pass that directly to playsound. simpler that way, and easier to extend

Reply:
will do